analysis/graphics.py

```diff
@@ -79,12 +106,13 @@ def create_second_figure(self):
             x1, y1 = [x1[i] + inc_x[i] for i in range(len(x1))], [y, y]
 
         # Add the division's lines for each column
-        for i in range(8):
+        for i in range(7):
```

Comment:
I do not remember this number, which is a good example of MAGIC NUMBER why did you reduce from 8 to 7 and what is the meaning?